tests/test_wyre_api.py

```diff
@@ -0,0 +1,194 @@
+from decimal import Decimal
+from urllib.parse import urljoin
+
+import pytest
+from django.conf import settings
+from rest_framework import status
+from stellar_sdk import Keypair
+
+from polaris_wyre.helpers.exceptions import WyreAPIError
+from polaris_wyre.wyre.api import TEST_BASE_URL
+from polaris_wyre.wyre.dtos import TransferData
+from .mocks import wyre as wyre_mocks
+
+
+REQUEST_METHOD_GET_MOCK = "requests.Session.get"
+REQUEST_METHOD_POST_MOCK = "requests.Session.post"
+
+
+def test_get_account_success(mocker, make_wyre_api):
+    wyre_request_mock = mocker.patch(
+        REQUEST_METHOD_GET_MOCK,
+        return_value=wyre_mocks.get_account_response(
+            account_id=settings.WYRE_ACCOUNT_ID
+        ),
+    )
+
+    wyre_api = make_wyre_api()
+    response = wyre_api.get_account()
+
+    wyre_request_mock.assert_called_once_with(
+        urljoin(wyre_api.API_URL, "v2/account"),
+    )
+
+    assert response == wyre_request_mock.return_value.json()
+    assert settings.WYRE_ACCOUNT_ID == response["id"]
```

Comment:
You could also assert the status code of the response here.

Reply:
@vinibs I renamed the response's variable name to response data. It gives us a better understanding of the test. What do you think?

Reply:
Yeah, I think that calling it `response_data` makes it more clear. But it's not relevant to you to also check the status code returned from the calls, beside of the response data?

Reply:
Sounds good!